pvlib/spectrum/mismatch.py

```diff
@@ -113,32 +130,125 @@ def get_am15g(wavelength=None):
     More information about reference spectra is found here:
     https://www.nrel.gov/grid/solar-resource/spectra-am1.5.html
 
+    See Also
+    --------
+    pvlib.spectrum.get_standard_spectrum : reads also the direct and
+      extraterrestrial components of the spectrum.
+
     References
     ----------
     .. [1] ASTM "G173-03 Standard Tables for Reference Solar Spectral
        Irradiances: Direct Normal and Hemispherical on 37° Tilted Surface."
-    '''
+    """  # noqa: E501
     # Contributed by Anton Driesse (@adriesse), PV Performance Labs. Aug. 2022
+    # modified by @echedey-ls, as a wrapper of spectrum.get_standard_spectrum
+    standard = get_standard_spectrum(wavelength, standard="ASTM G173-03")
+    return standard["global"]
 
-    pvlib_path = pvlib.__path__[0]
-    filepath = os.path.join(pvlib_path, 'data', 'astm_g173_am15g.csv')
 
-    am15g = pd.read_csv(filepath, index_col=0).squeeze()
+def get_standard_spectrum(wavelengths=None, *, standard="ASTM G173-03"):
```

Comment:
Any thoughts on renaming this to `get_reference_spectrum` (https://github.com/pvlib/pvlib-python/pull/2039#discussion_r1610482446)?

Reply:
My eyes only focused on the parameter name, sorry 😞